rsconf/component/raydata_scan_monitor.py

```diff
@@ -49,5 +50,8 @@ def internal_build_write(self):
 
     def _unit_env(self, z):
         return pkconfig.to_environ(
-            ["raydata.pkcli.*"], values={"raydata.pkcli.scan_monitor.db_dir": z.db_d}
+            ["raydata.pkcli.*"],
+            values=PKDict(copy.deepcopy(z)).pkupdate(
```

Comment:
The "z"  is [already be a copy](https://github.com/radiasoft/rsconf/pull/271/files#diff-f1d8f33eea3bde336b215690e4d2ef52f73ef109cd8a3ded3178f1be28b000c8L22) so there should be no need for a further copy. I'm not sure why this particular environment variable not be written in internal_build_compile.